parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -143,7 +144,7 @@ retrycmd_get_tarball() {
         if [ $i -eq $tar_retries ]; then
             return 1
         else
-            timeout 60 curl -fsSLv $url -o $tarball
+            timeout 60 curl -fsSLv $url -o $tarball 2>&1 | tee $CURL_OUTPUT >/dev/null | grep -E "^(curl:.*)|([eE]rr.*)$" && cat $CURL_OUTPUT
```

Comment:
2>&1 - to combine stderr and stdout
tee $CURL_OUTPUT >/dev/null - to only write output to file and not console because it shows up in VHD logs
grep -E "^(curl:.*)|([eE]rr.*)$" - regex to check for err/Err and curl:
cat the file if the above regex matches.

Reply:
updated!